tests/wpunit/Tribe/Utils/DateTest.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace Tribe\Utils;
+
+use Tribe__Date_Utils as Dates;
+
+
+class DateTest extends \Codeception\TestCase\WPTestCase {
+
+	/**
+	 * @test
+	 */
+	public function test_date_sort_with_array() {
+		$now          = '47448000';
+		$start_date   = '47448001';
+		$end_date     = '47448002';
+		$sorted_array = [ $now, $start_date, $end_date ];
+		$test_array   = [ $end_date, $now, $start_date ];
+
+		$sorted = Dates::sort( $test_array );
+
+		// Ensure we're comparing apples to apples here, instead of datetime objects to strings.
+		$sorted = array_map(
+			function( $date ) {
+				return $date->format( 'U' );
+			},
+			$sorted
+		);
+
+		$this->assertEquals( $sorted_array, $sorted, 'Dates should be sorted in the appropriate ascending order.' );
+	}
+	/**
+	 * @test
+	 */
+	public function test_date_sort_desc() {
+		$now        = '47448000';
```

Comment:
These tests are fine, but the `sort` method will accept an array of not just `int`, but `DateTime` objects and `strtotime` parsable strings: add tests that will use those inputs and a mix of them to make sure that support is actually working as intended.
Note: when using `DateTime`, use both the mutable and the immutable version of it (`DateTimeImmutable`) to make sure the handling of `DateTime` objects is done correctly.

Reply:
Addressed